packages/database/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@colonial-collections/database",
+  "version": "0.0.0",
+  "private": true,
+  "main": "./src/index.ts",
+  "types": "./src/index.ts",
+  "scripts": {
+    "db:generate": "dotenv -c -- drizzle-kit generate:mysql",
+    "db:migrate": "dotenv -c -- ts-node ./scripts/migrate.ts",
+    "db:drop": "dotenv -c -- drizzle-kit drop",
+    "db:studio": "dotenv -c -- drizzle-kit studio",
+    "db:create": "bash ./scripts/create-database.sh",
```

Comment:
Is the `bash` prefix required? If `create-database.sh` is executable (i.e. has the correct permissions), it should run without this prefix?